modules/distr-tracing-tempo-install-instance.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-instance_{context}"]
+= Installing a {TempoName} instance
+
+https://issues.redhat.com/browse/TRACING-3296
+
+Deploying the TempoStack
+
+You can use the OpenShift Container Platform web console to deploy the TempoStack.
+
+.Prerequisites
+
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the Tempo Operator:
+
+.. In the OpenShift Container Platform web console, click *Operators* → *OperatorHub*.
+
+.. Choose *Tempo Operator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces* on the cluster.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
```

Comment:
@rubenvp8510, in the OpenShift web console, the default preset for the Tempo Operator is *openshift-operators* rather than *openshift-operators-redhat*. Is it OK?